src/Symfony/Component/Intl/Tests/DateFormatter/Verification/IntlDateFormatterTest.php

```diff
@@ -45,6 +45,8 @@ public function testFormatWithTimezoneFromEnvironmentVariable()
 
     protected function getDateFormatter($locale, $datetype, $timetype, $timezone = null, $calendar = IntlDateFormatter::GREGORIAN, $pattern = null)
     {
+        IntlTestHelper::requireFullIntl($this, '55.1');
```

Comment:
I missed replacing the comparison operator from != to >=.
The intend is to make tests run more frequently instead of being skipped, especially on my Ubuntu LTS that ships with 55.1, and where tests run fine.
Having that many tests be skipped almost all the time and especially on our CI or even locally for most users means they fail to achieve their goal of preventing regressions.
This is an attempt to make the situation better.
Would be great if you could look into it in your PR also.